src/pages/news/2021-03-30-oplossen-kwetsbaarheden-podcast.md

```diff
@@ -0,0 +1,20 @@
+---
+path: "/news/2021-03-30-2021-03-30-oplossen-kwetsbaarheden-podcast/"
```

Comment:
typo filename is called "2021-03-30-oplossen-kwetsbaarheden-podcast/" this will result in a file not found and not render